Resources/Prototypes/Catalog/Fills/Crates/fun.yml

```diff
@@ -277,6 +277,17 @@
       - id: WeaponWaterPistol
         amount: 4
 
+- type: entity
+  id: CrateWaterBalloons
+  parent: CratePlastic
+  name: water balloons crate
+  description: Bring forth the splashing war with these silly balloons!
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxWaterBalloon
+        amount: 2
```

Comment:
```suggestion
    - id: BoxWaterBalloon
      amount: 2
```

Reply:
every other crate has the - aligned with the n in contents so i thinkie its oki